public_html/invalid_verification_link.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<!-- This page is displayed when someone navigates to a verify email or reset password link
```

Comment:
Do you mean removing the comments from the html file entirely or is there some kind of general solution the can remove comments from the file before being sent back? 

The existing invalid_link.html file already has similar set of comments (in fact, the exact same comments that appear in these files since I just copied over the contents of invalid_link and made a few alterations 😅 )